segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListener.java

```diff
@@ -156,6 +157,7 @@ public PravegaConnectionListener(boolean enableTls, boolean enableTlsReload, Str
         }
         this.replyWithStackTraceOnError = replyWithStackTraceOnError;
         this.connectionTracker = new ConnectionTracker();
+        this.maxReadLength = maxReadLength;
```

Comment:
Please add a `Preconditions.checkArgument` to validate that this is at least X KB. (X == 10?).